solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1278,6 +1287,12 @@ public Builder withMaxConnectionsPerHost(int max) {
       return this;
     }
 
+    /** The maximum outstanding async requests, default is 1000 requests */
```

Comment:
Follow standard javadoc convention here.

Reply:
oops I don't know why `gradlew tidy` do collapse the comment, let me update the comment to make it longer than 1 line

Reply:
afaict this is in fact standard javadoc formatting, judging by existing patterns for this project anyway. I've observed that it's not uncommon, at least for cases where  the arg and return value are obvious from the text description, for `@param` and `@return` to be omitted, and for shorter javadocs to be collapsed to a single line. No need to fight w/ `gradlew tidy` here 🙂